WindowsFunctionsStacks.json

```diff
@@ -131,15 +161,19 @@
                         "displayVersion": "11",
                         "runtimeVersion": "11",
                         "supportedFunctionsExtensionVersions": [
+                            "~2",
```

Comment:
We don't like to have support for java11 on v2 using tooling.

Reply:
so Java 11 only supported in V3 both on windows and linux?

Reply:
Java 11 only supported in V3 both on windows and linux from tooling yes.

Java 11 is still work on v2 windows but we should not support that in tooling, the only way this case can happens the user change javaversion value in the resource manager.

Reply:
got it. For the purpose of this file, we can state Java 11 only avaialble on V3. @gzuber  - please udpate.